crates/neon-sys/native/src/neon_class_metadata.h

```diff
@@ -20,7 +20,7 @@
 // Slots 0 and 1 appear to be reserved by Chrome, and slot 3 is reserved by Node.
 // That apparently leaves only slot 2 available for storing Neon metadata.
 //
-// See: https://github.com/nodejs/node/blob/master/src/env.h#L33-L39
+// See: https://github.com/nodejs/node/blob/main/src/env.h#L33-L39
```

Comment:
this one shouldnt be changed :D

```suggestion
// See: https://github.com/nodejs/node/blob/master/src/env.h#L33-L39
```

Reply:
🤦